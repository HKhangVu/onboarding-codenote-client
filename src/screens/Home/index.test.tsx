import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import Home from './index';
import store from '../../store';

test('renders home', () => {
    const wrapper = shallow(
        <Provider store={store}>
            <Home />
        </Provider>  
    );
    expect(wrapper).toMatchSnapshot();
});
