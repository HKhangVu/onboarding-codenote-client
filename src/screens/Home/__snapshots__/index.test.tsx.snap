// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home 1`] = `<Connect(Home) />`;
